src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -216,7 +218,12 @@ private void setHadoopBAMConfigurationProperties(final String inputName, final S
         final Configuration conf = ctx.hadoopConfiguration();
         conf.set(SAMHeaderReader.VALIDATION_STRINGENCY_PROPERTY, validationStringency.name());
 
-        if (!IOUtils.isCramFileName(inputName)) { // only set the reference for CRAM input
+        // If the input we're reading is sharded output, filePath is the name of a folder that contains
+        // the individual shard parts, and the original file extension (i.e., ".cram") is embedded in
+        // the middle of the name, so we need to scan the whole name to detect CRAM input
```

Comment:
It looks like this case was an artifact of the way the tests were previously written, but is no longer needed. So I'm going to remove the extra test.